src/doom/d_main.c

```diff
@@ -1614,6 +1614,31 @@ void D_DoomMain (void)
     // we've finished loading Dehacked patches.
     D_SetGameDescription();
 
+    //!
+    // @arg <directory>
+    //
+    // Specify a path from which to load and save games. If the directory
+    // does not exist then it will automatically be created.
+    // NOTE TO WINDOWS USERS: Do not end a path with a backslash ("\") if it
```

Comment:
Please remove this "NOTE TO WINDOWS USERS" - while it's well-intentioned, I don't think it's necessary.